autogen/agentchat/assistant_agent.py

```diff
@@ -15,9 +18,12 @@ class AssistantAgent(ConversableAgent):
     This agent doesn't execute code by default, and expects the user to execute the code.
     """
 
-    DEFAULT_SYSTEM_MESSAGE = """You are a helpful AI assistant.
+    shell_command = "sh" if sys.platform != "win32" else get_powershell_command()
```

Comment:
I am wondering if you can use a static variable here instead of calling the function. Isn't the static variable already in `code_utils`?

Reply:
yes sure

Reply:
done!